### DISCUSSION TITLE
What LORA training software do you use?

### DISCUSSION THREAD

**Discussion by msglm** [leejet/stable-diffusion.cpp#510] [General]
My favorite part of this project is that it's a single binary and thus very easy to compile and deploy. Most tools I've seen for training LORAs are very fragile and annoying to use. What do you use for making your LORAs?

**Comment by Amin456789** [comment]
please make a lora support if u can @stduhpf

**Reply by stduhpf** [reply]
You mean lora training?

**Reply by Amin456789** [reply]
if possible, yes

**Reply by nonetrix** [reply]
Would really like this I think it might work better on older AMD cards having a lot of issues with sd-scripts with something that even people with 12GBs of VRAM on NVIDIA are seemingly having better luck with

**Comment by ddpasa** [comment]
I use sd-scripts for SDXL and Flux1-dev lora training.  However, the project doesn't appear to be super active, and the devs seem to have moved to Musubi tuner: https://github.com/kohya-ss/sd-scripts/issues/2243

**Reply by nonetrix** [reply]
I'll have to try that one never heard of it, knowing my luck I'll still get OOMs on my RX 6800 even though it would work on any card that isn't AMD garbage
